platform/commonUI/edit/bundle.js

```diff
@@ -203,20 +205,30 @@ define([
                     ]
                 },
                 {
+                    "key": "save-and-stop-editing",
+                    "category": "save",
+                    "implementation": SaveAndStopEditingAction,
+                    "name": "Save and Done Editing",
```

Comment:
Actually, Pete has convinced me that Save and Finish is better than Save and Done. And, we can always tweak this later if needed. :)